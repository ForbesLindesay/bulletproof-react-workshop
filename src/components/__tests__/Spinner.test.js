// @flow

import React from 'react';
import renderer from 'react-test-renderer';
import Spinner from '../Spinner';

test('Spinner', () => {
  expect(renderer.create(<Spinner />).toJSON()).toMatchSnapshot();
});
