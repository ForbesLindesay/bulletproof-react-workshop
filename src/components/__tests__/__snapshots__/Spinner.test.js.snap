// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner 1`] = `
<div>
  Loading...
</div>
`;
